packages/babel-traverse/src/scope/index.js

```diff
@@ -830,7 +832,7 @@ export default class Scope {
         return scope;
       }
     } while ((scope = scope.parent));
-    throw new Error("We couldn't find a Function or Program...");
+    return null;
```

Comment:
I don't have strong feelings on this decision, but I do agree that the name and the behavior seem contradictory.

If we do go with this change, the doc comment on this function also needs to be updated.